README.md

```diff
@@ -422,6 +422,39 @@ done manually by someone on the Azul team. Ask on Slack to find out more.
 
 [Terra Data Repository]: https://jade.datarepo-dev.broadinstitute.org/
 
+### 3.2.2 SAM registration of Google accounts for developers
+
+While service accounts are associated with Azul, a developer Google account is 
+associated with a developer. For the remainder of this section developer Google 
+accounts will be referred to as your account. For example a developer at UCSC 
+would have an account ending in `ucsc.edu`.
+
+There are two reasons to register your account with SAM, to allow your account 
```

Comment:
I changed up the intro to this paragraph in e9e2d9af38ad962dfb5d48a5eeb4331e5185ebe3 . I thought the changes were less awkward.